# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openfodder/hotfix.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- openfodder-2.0.0/Makefile.vanilla	2025-12-19 12:53:00.940189221 +0100
+++ openfodder-2.0.0/Makefile	2025-12-19 12:53:21.203512010 +0100
@@ -9,11 +9,9 @@
 all : fodder
 
 fodder: 	main
-		mv *.o obj/
-		$(LD) -o Run/OpenFodder
 
 main:
-		git log -n 1 --pretty="const char* gitversion=\"%h\";" > ./Source/gitver.hpp
+		echo const char* gitversion=\"$(ver)\"\; > ./Source/gitver.hpp
 		$(CC) Source/*.cpp Source/PC/*.cpp Source/Amiga/*.cpp Source/Structures/*.cpp Source/Utils/*.cpp Source/Map/*.cpp
 		mkdir -p obj
 
